### PR TITLE
Bring the README up to date with the GUI Team info

### PR DESCRIPTION
**Pull request by olizilla**
The IPFS GUI Team is a thing. Let us honour it's official status
with a fresh README, updated project list, resources, and call out
the team members who've already started this adventure.

License: MIT
Signed-off-by: Oli Evans <oli@tableflip.io>

### CODE REVIEW

**Comment by olizilla** [comment]
@akrych @andreforsousa @alanshaw @fsdiogo @hacdias @lidel @meiqimichelle you are all name checked as part of the IPFS GUI Team. I have taken the liberty of introducing you. This is to honour your contributions, but please do feel free to say if you dislike it or would like to change your intro!

**Comment by olizilla** [comment]
@diasdavid the gui triangle has gone. It is beautiful, and it's teaching lives on in our hearts.

File: README.md [hunk 1]
```diff
@@ -1,21 +1,43 @@
...
+# The IPFS GUI Team
 
-> Make the IPFS GUIs simple, accessible, reusable and beautiful. That is the IPFS GUI Team! 
+> The meeting point for making great Graphical User Interfaces.
 
 ## Goal
 
 Make the IPFS GUIs simple, accessible, reusable and beautiful.
 
+- **Simple** - Fight complexity at every step. The are big ideas in IPFS. Let's present them clearly.
```
Comment (fsdiogo) [inline README.md]: `There` are big ideas in IPFS.

File: README.md [hunk 2]
```diff
@@ -1,21 +1,43 @@
...
+- **Accessible** - IPFS **must** be useable and comprehensible for everyone.
```
Comment (fsdiogo) [inline README.md]: `usable`

File: README.md [hunk 3]
```diff
@@ -1,21 +1,43 @@
...
+- **Reusable** - Publish and promote this work. Make doing the right thing the easist thing.
```
Comment (fsdiogo) [inline README.md]: `easiest`

File: README.md [hunk 4]
```diff
@@ -1,21 +1,43 @@
...
+- **Beautiful** - Demonstreate the nature of the system with effortless, coherent and compelling interfaces.
```
Comment (fsdiogo) [inline README.md]: `Demonstrate`

File: README.md [hunk 5]
```diff
@@ -25,46 +47,42 @@ From there, we’ll design wireframes for each app, composing them from the list
...
+- @hacdias - Desktop lead dev and Web UI - the file manager mage
+- @lidel - Companion lead dev and original IPFS sage
+- @meiqimichelle - Advisor and User-centered design expert
+- @olizilla - Web UI lead dev and GUI team gardener
+
+**and you...**
+
+## Contributing
+
+We figure things out in the open and coordinate via the [**issues on this repo**](https://github.com/ipfs/ipfs-gui/issues). You can **chat to us in irc** in the <a href="https://www.irccloud.com/invite?channel=%23ipfs-in-web-browsers&amp;hostname=irc.freenode.net&amp;port=6697&amp;ssl=1"> #ipfs-in-web-browsers channel on irc.freenode.net</a>. We're an open project and a friendly group, so be nice and **read the [contributing guidelines](https://github.com/ipfs/community/blob/master/contributing.md)** when you're ready to jump in.
```
Comment (fsdiogo) [inline README.md]: You can chat `with` us.

File: README.md [hunk 6]
```diff
@@ -25,46 +47,42 @@ From there, we’ll design wireframes for each app, composing them from the list
...
 
-- Design
-  - Assets in Google Drive (soon™)
-  - Zeplin
-    - [WebUI Wireframe](https://app.zeplin.io/project/5a32d45d1a17248135241058)
-  - [Design Tools Research](https://docs.google.com/document/d/1qJyfwgcMg8l3Tk3aYxF38iyYRhkEf3nlLNqOw4ZiW_8/edit)
+## Team
+
+In alphabetical order we are:
+
```
Comment (fsdiogo) [inline README.md]: Should we link these to the GitHub profiles? 🤔

Comment (olizilla) [reply README.md]: i considered it, but then left it. If people wanna creep around our profiles they can figure it out, but i don't feel strongly about it, happy to link them if people feel positive about it

File: README.md (hunk 2 above)
Comment (olizilla) [reply README.md]: oh my, i've been spelling that wrong for some time.

File: README.md (hunk 4 above)
Comment (olizilla) [reply README.md]: so many typos in so few lines!

File: README.md (hunk 6 above)
Comment (fsdiogo) [reply README.md]: Eh, don't have a strong opinion on that too 🤷‍♂️

**Comment by olizilla** [comment]
@andreforsousa @alanshaw @meiqimichelle I'm gonna ship this with your fancy names on it 
:shipit: ✨🚢

**Pull request merged**